server/routes_test.go

```diff
@@ -0,0 +1,76 @@
+package server
+
+import (
+	"context"
+	"io"
+	"net/http"
+	"net/http/httptest"
+	"testing"
+
+	"gotest.tools/v3/assert"
```

Comment:
testify is more standard, but this is fine too
https://github.com/stretchr/testify

Reply:
I switched over to testify. I don't have a preference, but we had used gotest.tools before.